src/components/Facility/FacilityHome.tsx

```diff
@@ -105,23 +106,24 @@ export const FacilityHome = ({ facilityId }: Props) => {
       pathParams: { id: facilityId },
     }),
   });
+  const { mutate: deleteFacility } = useMutation({
+    mutationFn: () =>
+      mutate(routes.deleteFacility, { pathParams: { id: facilityId } })(null),
+    onSuccess: () => {
+      toast.success(t("deleted_successfully", { name: facilityData?.name }));
+      navigate("/facility");
+    },
+    onError: () => {
+      toast.error(t("delete_failed", { name: facilityData?.name }));
+    },
```

Comment:
remove all onErrors on other places as well (that are only toasting, and not setting any form states)
```suggestion
```